webapp/store/survey/actions/actionTypes.js

```diff
@@ -3,3 +3,5 @@ export const surveyUpdate = 'survey/update'
 export const surveyDelete = 'survey/delete'
 export const surveyDefsLoad = 'survey/defs/load'
 export const surveyDefsReset = 'survey/defs/reset'
+export const surveyChainSave = 'survey/chain/save'
+export const surveyChainElementDelete = 'survey/chain/delete'
```

Comment:
this is better: ```export const surveyChainItemDelete = 'survey/chain/item/delete'```

Reply:
Thanks, I had some bad feelings when I put that name